src/App.js

```diff
@@ -33,14 +33,18 @@ function App() {
     <Router>
       <div>
         <Navbar className="Nav" bg="dark" variant="dark">
-          <Navbar.Brand href="/" onClick={(e) => e.preventDefault()}>chatMuch Lite</Navbar.Brand>
+          <Navbar.Brand href="/" onClick={(e) => {alert('All Sign-data will be lost');}} >chatMuch Lite</Navbar.Brand>
```

Comment:
I think you could wrap this in a `<Link>` tag to prevent it from reseting state.